modules/http/src/main/java/org/mule/extension/http/internal/request/MuleEventToHttpRequest.java

```diff
@@ -96,15 +98,15 @@ public MuleEventToHttpRequest(HttpRequesterConfig config, String uri, String met
   public HttpRequest create(Event event, HttpRequesterRequestBuilder requestBuilder, HttpAuthentication authentication,
                             MuleContext muleContext)
       throws MuleException {
-    HttpRequestBuilder builder = new HttpRequestBuilder();
+    HttpRequestBuilder builder = HttpRequest.builder();
```

Comment:
static imports? Would it be not clear or is there any other reason to avoid having an static import for this case?